### PR TITLE
Update the Japanese file to v3.12.1

### PR DESCRIPTION
**Pull request by hrwatahiki**
Update the Japanese file to v3.12.1

### CODE REVIEW

File: src/translations/sqlb_ja.ts [hunk 1]
```diff
@@ -4028,27 +4029,27 @@ SQL文をスキーマカラムからSQLエディターや他のアプリケー
...
         <location filename="../MainWindow.cpp" line="336"/>
         <source>Simplify Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>ウィンドウレイアウトをシンプルに</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="337"/>
         <source>Shift+Alt+0</source>
-        <translation type="unfinished"></translation>
+        <translation>Shift+Alt+0</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_ja.ts:4037]: Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.

File: src/translations/sqlb_ja.ts [hunk 2]
```diff
@@ -6080,142 +6081,142 @@ DB Browser for SQLite 設定のリモートタブに行き、新しい証明書
...
         <location filename="../RemoteDock.ui" line="411"/>
         <source>Reload all data and update the views</source>
-        <translation type="unfinished"></translation>
+        <translation>全てのデータを再読み込みしビューを更新する</translation>
     </message>
     <message>
         <location filename="../RemoteDock.ui" line="414"/>
         <source>F5</source>
-        <translation type="unfinished"></translation>
+        <translation>F5</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_ja.ts:6164]: Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.

**Review by lucydodo — CHANGES_REQUESTED** [review]
Thank you for contributing! :smile: 
I found two items that needed simple corrections.
I could only edit those parts and merge them, but I left it as a review only as you may have disagreements. Check them please.

**Comment by hrwatahiki** [comment]
@lucydodo 
Sorry. I removed translated shortcuts you quoted.

**Comment by lucydodo** [comment]
Thanks! Now squash and merged.

**Comment by hrwatahiki** [comment]
Thanks! 😊

**Pull request merged**